codewind-che-sidecar/tests/cronjob-sidecar.sh

```diff
@@ -60,6 +60,7 @@ fi
 
 export CHE_INGRESS_DOMAIN=che-$NAMESPACE.$CLUSTER_IP.nip.io
 export CHE_NAMESPACE=$NAMESPACE
+export CLUSTER_IP
```

Comment:
We are just setting this as an env variable in the crontab right? Is this needed here?

Reply:
Yes it is, since it is not set as a global env var in crontab so need to export it, so that child bash scripts spawned can inherit it